update-scripts/nvim-lspconfig/locate-lsp-packages.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python3
+# This script requires nix-locate
+
+import json
+import os
+import subprocess
+
+repo = os.path.realpath(os.path.dirname(__file__) + "/../..")
```

Comment:
Maybe this is why? I think there'd still be work-arounds, such as providing the relevant files at the CLI or via env vars. These could then be made available to the script via a wrapper or a patch/fixup phase in the drv builder.